backend/typescript/models/scheduling.model.ts

```diff
@@ -67,7 +67,7 @@ export default class Scheduling extends Model {
 
   @AllowNull(false)
   @Column({
-    type: DataType.ENUM("One time", "Daily", "Weekly", "Monthly"),
+    type: DataType.ENUM("One time donation", "Daily", "Weekly", "Monthly"),
```

Comment:
can we revert this change (+ the other ones)? there's more places where it's just "One Time" in the platform so you can just map it in the first scheduling page instead of changing it entirely
```suggestion
    type: DataType.ENUM("One time", "Daily", "Weekly", "Monthly"),
```